numpy/core/src/umath/ufunc_object.c

```diff
@@ -1025,6 +1029,13 @@ get_ufunc_arguments(PyUFuncObject *ufunc,
             }
 
             switch (str[0]) {
+                case 'a':
```

Comment:
Probably speed – I haven't checked if this particular case makes a meaningful difference, but the ufunc call path is a place where microseconds count.